### PR TITLE
🎨Storage: disallow optional S3/postgres settings

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- it makes no sense to run the storage service without S3 and postgres settings
- added validation of settings

## Related issue/s

- fixes https://github.com/ITISFoundation/osparc-simcore/issues/5680

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/storage/.env-devel [hunk 1]
```diff
@@ -23,6 +23,7 @@ S3_BUCKET_NAME=simcore
 # 172.17.0.1 is the docker0 interface, which redirect from inside a container onto the host network interface.
 S3_ENDPOINT=http://172.17.0.1:9001
 S3_SECRET_KEY=12345678
+S3_REGION=us-east-1
```
Comment (pcrespov) [inline services/storage/.env-devel:26]: THOUGHT: having dot-env files per service and globally (i.e. for the docker-compose specs) is redundant. This is the result of two different strategies that never converged.  These dot-env per service are only used for testing or local development. E.g. in `mock_environment` fixture for testing, or to start locally the service (probably not done anymore)

We could go two ways:
1. have a dot-env per service and merge them into one for the compose specs (probably requires also a `common` listing for env vars shared)
2. have a global dot-env and every service parses what it is needed.

The second approach seems more feasible for me.

**Review by pcrespov — APPROVED** [review]
thx

File: services/storage/.env-devel (hunk 1 above)
Comment (sanderegg) [reply services/storage/.env-devel:26]: also prefer second option.

**Review by GitHK — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]
:pray:

**Review by wvangeit — APPROVED** [review]
Thanks a lot

**Pull request merged**